Formula/gdcm.rb

```diff
@@ -19,17 +23,39 @@ class Gdcm < Formula
 
   def install
     ENV.cxx11
+    ENV.append "LDFLAGS", "-undefined dynamic_lookup"
+
+    args = std_cmake_args + %w[
+      -DGDCM_BUILD_APPLICATIONS=ON
+      -DGDCM_BUILD_SHARED_LIBS=ON
+      -DGDCM_BUILD_TESTING=OFF
+      -DGDCM_BUILD_EXAMPLES=OFF
+      -DGDCM_BUILD_DOCBOOK_MANPAGES=OFF
+      -DGDCM_USE_VTK=ON
+      -DGDCM_USE_SYSTEM_OPENSSL=ON
+    ]
 
     mkdir "build" do
-      system "cmake", "..", "-DGDCM_BUILD_APPLICATIONS=ON",
-                            "-DGDCM_BUILD_SHARED_LIBS=ON",
-                            "-DGDCM_BUILD_TESTING=OFF",
-                            "-DGDCM_BUILD_EXAMPLES=OFF",
-                            "-DGDCM_BUILD_DOCBOOK_MANPAGES=OFF",
-                            "-DGDCM_USE_VTK=ON",
-                            "-DGDCM_USE_SYSTEM_OPENSSL=ON",
-                            *std_cmake_args
-      system "make", "install"
+      if build.without?("python") && build.without?("python3")
+        system "cmake", "..", *args
+        system "make", "install"
+      else
+        Language::Python.each_python(build) do |python, version|
+          python_include = `#{python} -c 'from distutils import sysconfig;print(sysconfig.get_python_inc(True))'`.chomp
+          python_args = %W[
```

Comment:
args +=

Reply:
But then `args` will become polluted when bindings are built for multiple python versions.

Reply:
You're right. Let's call the top variable common_args and then you can do args = common_args + %W[ here.